tools/setup_playwright.py

```
@@ -0,0 +1,42 @@
+import subprocess
+import sys
+from pathlib import Path
+
+def main():
+    """Install Playwright and Chromium browser."""
+    print("Setting up Playwright for HTML→PNG rendering...")
+    
+    try:
+        # Install Playwright browsers (Chromium)
+        print("Installing Chromium browser...")
+        result = subprocess.run([
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)